spec/validator_spec.js

```diff
@@ -50,19 +50,41 @@ describe("XMLParser", function() {
     });
 
     it("should validate self closing tags", function() {
-        const xmlData = "<rootNode><validtag1  /><validtag2/><validtag3  with='attrib'/></rootNode>";
+        const xmlData = "<rootNode><validtag1  /><validtag2/><validtag3  with='attrib'/><validtag4 />text<validtag5/>text</rootNode>";
         const result = validator.validate(xmlData);
         expect(result).toBe(true);
     });
 
-    it("should not validate self closing tags", function() {
-        const xmlData = "<rootNode><validtag1/><invalid tag/><validtag3  with='attrib'/></rootNode>";
-        const expected = {code: "InvalidAttr", msg: "boolean attribute tag is not allowed."};
+    it("should not consider these as self closing tags", function() {
+        let xmlData = "<rootNode><validtag1/><invalid tag/><validtag3  with='attrib'/></rootNode>";
+        let expected = {code: "InvalidAttr", msg: "boolean attribute tag is not allowed."};
 
-        const result = validator.validate(xmlData).err;
+        let result = validator.validate(xmlData).err;
         //console.log(JSON.stringify(result,null,4));
         expect(result).toEqual(expected);
+
+        xmlData = "<rootNode><notSelfClosing/ ></rootNode>";
+        expected = {code: "InvalidAttr", msg: "attribute / has no space in starting."};
+
+        result = validator.validate(xmlData).err;
+        expect(result).toEqual(expected);
+    });
+
+    /*
```

Comment:
This test is commented out because it doesn't pass due to another bug. This is addressed in https://github.com/NaturalIntelligence/fast-xml-parser/pull/205